deploy/twingate-operator/values.yaml

```diff
@@ -127,6 +127,11 @@ priorityClassName: ""
 extraEnvVars: []
 
 
+pdb:
+  create: false
+  # minAvailable: 1
+  # maxUnavailable: 
```

Comment:
Missing value?

Reply:
Also I just noticed other string properties like `nameOverride: ""` uses empty string as the template so using `""` would be more consistent with the rest of the file.